testdata/bp-all-with-fips.out.diff

```diff
@@ -61,58 +52,21 @@
   			},
   		},
   		Files: []blueprint.FileCustomization{
-  			{
-  				Path:  "/etc/systemd/system/custom.service",
-- 				User:  nil,
-+ 				User:  string("root"),
-- 				Group: nil,
-+ 				Group: string("root"),
-- 				Mode:  "",
-+ 				Mode:  "644",
-  				Data:  "[Unit]\nDescription=Custom service\n\n[Service]\nExecStart=/usr/bin/"...,
-  			},
-  			{
-  				Path:  "/etc/systemd/system/custom.service.d/override.conf",
-- 				User:  nil,
-+ 				User:  string("root"),
-- 				Group: nil,
-+ 				Group: string("root"),
-- 				Mode:  "",
-+ 				Mode:  "644",
-  				Data:  "[Service]\nExecStart=\nExecStart=/usr/bin/cat /etc/custom_file.txt\n",
-  			},
+  			{Path: "/etc/systemd/system/custom.service", Data: "[Unit]\nDescription=Custom service\n\n[Service]\nExecStart=/usr/bin/"...},
+  			{Path: "/etc/systemd/system/custom.service.d/override.conf", Data: "[Service]\nExecStart=\nExecStart=/usr/bin/cat /etc/custom_file.txt\n"},
   			{
   				Path:  "/etc/custom_file.txt",
   				User:  string("root"),
   				Group: string("root"),
 - 				Mode:  "0644",
-+ 				Mode:  "644",
++ 				Mode:  "",
   				Data:  "image builder is the best",
   			},
-  			{
-  				Path:  "/etc/empty_file.txt",
-  				User:  int64(0),
-  				Group: int64(0),
-- 				Mode:  "",
-+ 				Mode:  "644",
-  				Data:  "",
-  			},
-  		},
-  		Repositories: []blueprint.RepositoryCustomization{
-  			{
-  				... // 8 identical fields
-  				GPGCheck:       &true,
-  				RepoGPGCheck:   &false,
-- 				SSLVerify:      nil,
-+ 				SSLVerify:      &true,
-  				ModuleHotfixes: nil,
-  				Filename:       "",
-  				InstallFrom:    false,
-  			},
+  			{Path: "/etc/empty_file.txt", User: int64(0), Group: int64(0)},
   		},
-  		FIPS:      &true,
-  		Installer: nil,
-  		... // 4 identical fields
+  		Repositories: {{Id: "example", BaseURLs: {"https://example.com/download/yum"}, GPGKeys: {"-----BEGIN PGP PUBLIC KEY BLOCK-----\n\nmQGiBGRBSJURBACzCoe9UNfxOU"...}, Name: "Example repo", ...}},
+  		FIPS:         &true,
+  		... // 5 identical fields
   	},
   	Distro:  "",
   	Arch:    "",
```

Comment:
If this is your first review, please note that out.diff is a Go struct diff format coming out of the google-cmp package. It is diff-like output of a round-trip conversion of UBP>BP or vice versa. It can be hard to read.

The convertors are not able to convert all corner cases, so it exists so we "document" what else is missing. For example the file mode is written in "0644" but after conversion it is "644". All the files prefixed with `bp-` are imported from the `images` library without changes, we can fix these "problems" later.

Sometimes there is a diff hunk of something convertor simply does not / cannot convert.